Sources/SQLKit/Concurrency/SQLQueryFetcher+Concurrency.swift

```diff
@@ -2,26 +2,40 @@ import NIOCore
 
 public extension SQLQueryFetcher {
     func first<D>(decoding: D.Type) async throws -> D? where D: Decodable {
-        return try await self.first(decoding: D.self).get()
+        try await self.first(decoding: D.self).get()
     }
     
-    func first() async throws -> SQLRow? {
-        return try await self.first().get()
+    func first() async throws -> (any SQLRow)? {
+        try await self.first().get()
     }
     
     func all<D>(decoding: D.Type) async throws -> [D] where D: Decodable {
-        return try await self.all(decoding: D.self).get()
+        try await self.all(decoding: D.self).get()
     }
     
-    func all() async throws -> [SQLRow] {
-        return try await self.all().get()
+    func all() async throws -> [any SQLRow] {
+        try await self.all().get()
     }
     
+    #if swift(>=5.7)
+    @preconcurrency
```

Comment:
Why do we need this here?

Reply:
Truthfully? I don't actually know, I've just been mimicking what NIO does when they add `@Sendable` to closure parameters.

Reply:
I'm pretty sure we can remove these instances then. They're mainly used to silence warnings and I can't see anything here that would trigger a warning

Reply:
I'm pretty sure we can remove these instances then. They're mainly used to silence warnings and I can't see anything here that would trigger a warning